lib/app_profiler/railtie.rb

```diff
@@ -37,6 +43,15 @@ class Railtie < Rails::Railtie
       end
     end
 
+    initializer "app_profiler.enable_server" do
+      if AppProfiler.server.enabled
```

Comment:
Should this server starts when running tests, running migrations or starting the console? Or only when the Rails servers starts? Right now it starts every time.

Reply:
It should be enabled for any long-running process, so rails server or jobs.

What is the most idiomatic way to achieve this?